warehouse/templates/email/password-reset-unverified/body.html

```diff
@@ -0,0 +1,31 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/body.html" %}
+
+{% block content %}
+  <p>
+    {% trans %}Someone, perhaps you, has made a password reset request for a PyPI account associated with this email address.{% endtrans %}
+  </p>
+  <p>
+    {% trans %}However, the email used to make this request is not verified. You must verify your email address before you can reset your password.{% endtrans %}
+  </p>
+  <p>
+    {% trans href=request.route_url('help', _anchor='account-recovery') %}Follow <a href="{{ href }}">account recovery steps</a> for your PyPI account if you are unable to verify your email address.{% endtrans %}
+  </p>
```

Comment:
If the user has forgotten their password, they probably can't verify their email address, so telling them to verify their email isn't very helpful. This should probably say something like "if there is another email associated with your account, you should attempt a password reset via that email instead, otherwise you will need to follow the account recovery process."